src/scenes/ObservationsRouter/BiomassMeasurementsDetails.tsx

```diff
@@ -142,6 +150,26 @@ export default function BiomassMeasurementsDetails(): JSX.Element {
     (photo) => photo.type === 'Plot' && photo.position !== undefined && photo.position !== 'SouthwestCorner'
   );
 
+  const unrecognizedSpecies: string[] = useMemo(() => {
```

Comment:
Don't we need to show the page message as in the other pages when there are unrecognized species?

Reply:
Ah, good point. The Jira task didn't mention that so I didn't think of it. Will add it.